metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/MathFunctions.java

```diff
@@ -60,4 +60,38 @@ public boolean isInitialized() {
     }
   }
 
+  @Stellar(name="POWER"
```

Comment:
I went with POWER to be consistent with SQL and Excel, rather than python, since that's a more likely familiar ground for security and BI people.